stack.yaml.lock

```diff
@@ -5,57 +5,15 @@
 
 packages:
 - completed:
-    hackage: binary-0.8.7.0@sha256:ae3e6cca723ac55c54bbb3fa771bcf18142bc727afd57818e66d6ee6c8044f12,7705
+    hackage: happstack-server-7.6.1@sha256:639af7840c897a3a0273bb183149990cbf18127077a025c3004606e27cbe56a0,6028
```

Comment:
the stack.yaml.lock changes are still showing up. let me know if you need help with reverting this to its original state!